Cargo.toml

```
@@ -13,6 +13,8 @@ bitcoin = "0.28.1"
 lightning = { version = "0.0.110", features = ["max_level_trace"] }
 log = "0.4.17"
 rand = "0.8.5"
+reqwest = { version = "0.11", features = ["json", "socks"] }
```

Change the `reqwest` features so that we can build for other platforms. 

```suggestion
reqwest = { version = "0.11", default-features = false, features = ["json", "socks", "rustls-tls"] }
```